app/walletButton.tsx

```diff
@@ -16,17 +16,28 @@ import Button from "@/app/button";
 export default function WalletButton() {
   const [modal, setModal] = useState<undefined | { type: string }>();
   const [bitcoinWallet, setBitcoinWallet] = useAtom(atomBitcoinWallet);
+
   function onClick() {
     if (bitcoinWallet) {
       setModal({ type: "bitcoinWallet" });
-      return;
+    } else {
```

Comment:
my golang-inspired preference is to not use else statements and to write these conditionals like this:

```
if (bitcoinWallet) {
    setModal({ type: "bitcoinWallet" });
    return;
}
setModal({ type: "selectWallet" });
```

Where the indentation represents the off-nominal case.